app/client/src/utils/arcGisRestUtils.ts

```diff
@@ -1173,6 +1173,19 @@ export async function addWebMap({
           } else {
             subLayers.push({
               id: subLayer.layerId,
+              disablePopup: false,
+              popupInfo: {
+                popupElements: [
+                  {
+                    type: 'fields',
+                    description: '',
+                    fieldInfos: popupFields,
+                    title: '',
+                  },
+                ],
+                fieldInfos: popupFields,
+                title: `${subLayer.title}: {orgName}`,
```

Comment:
Good catch! This should be fixed now. 

I restructured our `layerProps.json` file (S3 bucket) a little bit. The `layerProps` key is everything that can be directly sent to AGO for creating the layer and the `popupTitle` is what I'm using to fill in the popup titles with arcade expressions. In the future we can add additional non-AGO props at the same level as `popupTitle`. This will be more flexible and allow us to change the titles without a full release. 

I also added popups to more of our supporting feature layers (i.e. counties, wild/scenic rivers, etc.). 

I left off supporting map image layers, because they are a bit more difficult. We would have to query the layer and all of its sublayers and then piece together the popup fields from that. I think we could eventually get these in there, I might create a separate ticket for that. I don't attempt to add popups to any of the layers added via the `Search` and `URL` tabs of the add data widget, because we want to preserve those layers exactly as they are provided.